vscode/src/extension.ts

```diff
@@ -519,17 +518,20 @@ export function activate(context: ExtensionContext): VSNetBeansAPI {
         }
         const userDir = path.join(storagePath, "userdir");
         if (userDir && fs.existsSync(userDir)) {
-            const confirmation = await vscode.window.showInformationMessage('Are you sure you want to delete cache for this workspace?',
-                'Yes', 'Cancel');
-            if (confirmation === 'Yes') {
+            const yes =  l10n.getLocaleValue("jdk.extension.cache.label.confirmation.yes")
+            const cancel = l10n.getLocaleValue("jdk.extension.cache.label.confirmation.cancel")
+            const confirmation = await vscode.window.showInformationMessage(l10n.getLocaleValue("jdk.extension.cache.message.confirmToDeleteCache"),
+            yes, cancel);
+            if (confirmation === yes) {
                 await fs.promises.rmdir(userDir, {recursive : true});
-                const res = await vscode.window.showInformationMessage('Cache cleared successfully for this workspace', 'Reload window');
-                if (res === 'Reload window') {
+                const reloadWindowActionLabel = l10n.getLocaleValue("jdk.extension.cache.label.reloadWindow");
+                const res = await vscode.window.showInformationMessage(l10n.getLocaleValue("jdk.extension.cache.message.cacheCleared"), reloadWindowActionLabel);
+                if (res === reloadWindowActionLabel) {
                     await vscode.commands.executeCommand('workbench.action.reloadWindow');
                 }
             }
         } else {
-            vscode.window.showErrorMessage('Cannot find userdir path');
+            vscode.window.showErrorMessage(l10n.getLocaleValue("jdk.extension.cache.message.noUserDir"));
         }
     }));
 
```

Comment:
Should the errors/prompts for actions also be localised?
Examples:
- line 432: `Client ${c} doesn't support new from template`
- line 454: `Client ${c} doesn't support new project`
- line 509: `Client ${c} doesn't support go to test`
- line 516: `'Cannot find workspace path'`
- line 540: `'Compiling workspace...'`
- line 543: `'Cleaning workspace...'`
- line 546: `'Compiling...'`
- line 549: `'Cleaning...'`
- line 552: `'Opening type...'`
- line 565: `'No super implementation found'`
- line 577: `'Surround with ...'`

Reply:
there are labels for some of these translated but somehow i have  not added them in code, pushing changes